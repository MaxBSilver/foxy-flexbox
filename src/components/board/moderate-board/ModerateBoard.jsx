import React from 'react';

export default function ModerateBoard(props) {
    const direction = props.flexDirection;
    const foxAlign = props.userGuessTwo;
    const boxAlign = props.alignContent;
    const foxJustify = props.userGuessOne;
    const boxJustify = props.justifyContent;
    return (
    <section className = "board--container">
        <section className ="fox--container" style = {{justifyContent : foxJustify, alignItems : foxAlign, flexDirection: direction}}>
            <article className = "fox"><img className = "fox-img" alt="fox" src="https://ya-webdesign.com/images/kawaii-cats-png-6.png"></img></article>
            <article className = "fox"><img className = "fox-img" alt="fox" src="https://ya-webdesign.com/images/kawaii-cats-png-6.png"></img></article>
        </section>
        <section className ="box--container" style = {{justifyContent : boxJustify, alignItems : boxAlign , flexDirection: direction}}>
            <article className = "box"></article>
            <article className = "box"></article>
        </section>
    </section>
  )
}
