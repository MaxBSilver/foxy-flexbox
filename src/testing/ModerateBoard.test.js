import React from 'react';
import ModerateBoard from '../components/answer/Answer';
import { shallow } from 'enzyme';
describe('ModerateBoard', () => {
    let wrapper;
  
    beforeEach(() => {
      wrapper = shallow(
        <ModerateBoard
        />
      );
    });
    it('should match the snapshot with all data passed in', () => {
        expect(wrapper).toMatchSnapshot();
      });
  });