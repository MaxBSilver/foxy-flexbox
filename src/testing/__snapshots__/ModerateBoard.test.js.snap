// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModerateBoard should match the snapshot with all data passed in 1`] = `
<section
  className="board--container"
>
  <section
    className="fox--container"
    style={
      Object {
        "alignItems": undefined,
        "flexDirection": undefined,
        "justifyContent": undefined,
      }
    }
  >
    <article
      className="fox"
    >
      <img
        alt="fox"
        className="fox-img"
        src="https://i.imgur.com/XnHxBnB.png"
      />
    </article>
    <article
      className="fox"
    >
      <img
        alt="fox"
        className="fox-img"
        src="https://i.imgur.com/FmUqM1R.png"
      />
    </article>
    <article
      className="fox"
    >
      <img
        alt="fox"
        className="fox-img"
        src="http://clipart-library.com/img/749202.png"
      />
    </article>
  </section>
  <section
    className="box--container"
    style={
      Object {
        "alignItems": undefined,
        "flexDirection": undefined,
        "justifyContent": undefined,
      }
    }
  >
    <article
      className="box"
    />
    <article
      className="box"
    />
    <article
      className="box"
    />
  </section>
</section>
`;
